### DISCUSSION TITLE
Image Upload 500 Server Error

### DISCUSSION THREAD

**Discussion by ghost** [thedevdojo/voyager#5393] [Q&A]
## Version information

 - Laravel: v8
 - Voyager: v1.4
 - PHP:7.4.20
 - Database: [MySQL] [8.0] (e.g. MySQL 8.0)

## Steps To Reproduce
1. Go to 'domain/admin'
2. Click on 'user/edit' And 'post image upload'
4. 500 Server

## Screenshots
https://ibb.co/hfY3hzZ

**Comment by MrCrayon** [comment]
You need to check your server logs and debug from there.
99.9% the problem is related to your server configuration and not Voyager.
